### PR TITLE
Spanish translations of ingredients taxonomy

### PR DESCRIPTION
**Pull request by AcuarioCat**
Translations for unknown

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -54384,6 +54440,10 @@ sv:vaniljstång
 carbon_footprint_fr_foodges_ingredient:fr:Gousse de Vanille
 carbon_footprint_fr_foodges_value:fr:2.6
 
+<en:vanilla
+en:vanilla vein
+es: vaina de vainilla, vainas de vainilla
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt:54445]: Those should go as synonyms of this entry:

<en:vanilla
en:vanilla pod, vanilla pods
de:Vanilleschoten
es:vainas de vainilla

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -63653,12 +63713,15 @@ nl:koffie van biologische oorsprong
...
 <en:coffee
+en:decaffeinated coffee
+es:cafe descafeinado
 fr:café décaféiné
 fi:kofeiiniton kahvi
 nl:Cafeïnevrije koffie
 # ingredient/fr:cafe-decafeine has 8 products @2019-06-03
 
 <en:instant coffee
+es:cafe instantáneo
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt:63724]: we already have it here:

<en:coffee
en:instant coffee, soluble Coffee
ar:ﻖﻫﻭﺓ ﻑﻭﺮﻳﺓ
az:Hazır qəhvə
bg:Разтворимо кафе
ca:cafè instantani
cs:Instantní káva
de:löslicher Kaffee
el:Στιγμιαίος καφές
eo:tuja kafo
es:café soluble, café instantáneo

File: taxonomies/ingredients.txt [hunk 3]
```diff
@@ -65821,7 +65884,7 @@ fr:extraits naturels
 <en:extract
 en:vegetable extract, vegetable extracts
 de:Pflanzenextrakt, Pflanzenextrakte, Gemüseextrakte
-es:Extracto vegetal
+es:Extracto vegetal, extratos vegetales
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt:65887]: is it extracto or extrato ? the singular is different than the plural

File: taxonomies/ingredients.txt (hunk 1 above)
Comment (AcuarioCat) [reply taxonomies/ingredients.txt:54445]: ok

File: taxonomies/ingredients.txt (hunk 2 above)
Comment (AcuarioCat) [reply taxonomies/ingredients.txt:63724]: ok, I thought I saw it in the list of unknowns - when It's next processed I'll check

File: taxonomies/ingredients.txt (hunk 3 above)
Comment (AcuarioCat) [reply taxonomies/ingredients.txt:65887]: Yes it is - I just added the plural to what was there but that was wrong.. I'll change it

**Pull request merged**